routes/cube_routes.js

```diff
@@ -1556,6 +1556,123 @@ router.get('/download/plaintext/:id', async (req, res) => {
   }
 });
 
+function shuffle(a) {
+  for (let i = a.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [a[i], a[j]] = [a[j], a[i]];
+  }
+  return a;
+}
+
+router.post('/startsealed/:id', body('packs').toInt({ min: 1, max: 16 }), body('cards').toInt(), async (req, res) => {
+  try {
+    const user = await User.findById(req.user);
+
+    if (!user) {
+      req.flash('danger', 'Please Login to build a sealed deck.');
+      return res.redirect(`cube/playtest/${req.params.id}`);
+    }
+    const params = {
```

Comment:
Why is this an object? I assume just because we're copying-and-pasting but not necessary.
```suggestion
    const packs = parseInt(req.body.packs, 10);
    const cards = parseInt(req.body.cards, 10);
```